datasets/standvastgoed/vastgoed_0.0.4.json

```diff
@@ -0,0 +1,477 @@
+{
+  "type": "dataset",
+  "id": "standVastgoed",
+  "version": "0.0.4",
+  "status": "beschikbaar",
+  "auth": "FP/MDW",
+  "title": "stand van de vastgoedgegevens",
+  "description": "stand op één bepaalde peildatum van gegevens in het vastgoed domein",
+  "publisher": "Onderzoek en Statistiek, team Basisstatistiek",
+  "contactPoint" : {
+    "vcard:fn" : "Basisstatistiek",
+    "vcard:hasEmail" : "basisstatistiek.ois@amsterdam.nl"
+  },
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "gebouwen",
+      "type": "table",
+      "version": "1.0.0",
+      "description": "Alle adresseerbare objecten (verblijfsobjecten, lig- en standplaatsen) met hun kenmerken en gebiedsindeling",
+      "temporal": {
+        "identifier": "nummeraanduidingVolgnummer",
+        "dimensions": {
+          "geldigOp": [
+            "standBeginGeldigheid",
+            "standEindGeldigheid"
+          ]
+        }
+      },
+      "schema": {
+        "$id": "https://github.com/Amsterdam/schemas/standvastgoed",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "identifier": [
+          "id"
+        ],
+        "required": [
+          "schema",
+          "id"
+        ],
+        "display": "id",
+        "properties": {
```

Comment:
It is best practice for each field to add a `title` and `description`, see [here](https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#permitted-attributes). Could you add these attributes for each field?

Reply:
heb ik overgenomen van https://github.com/Amsterdam/amsterdam-schema/blob/master/datasets/bag/panden/v1.1.0.json#L34.